### ISSUE TITLE
Homepage: movie card hover effect

### ISSUE THREAD

**Issue by okalimbet** [Shawntru/Rotten-Tomatillos#5]
Each movie card should have a hover effect 

that allows users to see which movie

they are selecting or about to choose.
